src/components/textField/presets/outline.ts

```diff
@@ -14,5 +15,6 @@ const styles = StyleSheet.create({
 
 export default {
   ...underline,
+  clearButtonStyle: styles.clearButtonStyle,
```

Comment:
You get this from the underline speared, so if you don't change it here you can remove this line